cryptofeed/exchanges/mixins/bitfinex_rest.py

```diff
@@ -194,3 +206,66 @@ async def candles(self, symbol: str, start=None, end=None, interval='1m', retry_
             if not end or len(data) < 10000:
                 break
             start = data[-1].start + offset
+
+    # Trading APIs
+
+    async def _post_private(self, endpoint: str, payload=None, api=None):
+        if not payload:
+            payload = {}
+        query_string = json.dumps(payload)
+        if not api:
+            api = self.auth_api
+        url = f'{api}/{endpoint}'
+        headers = self._generate_signature(endpoint, query_string)
+        data = await self.http_conn.write(url, msg=query_string, header=headers)
+        return json.loads(data, parse_float=Decimal)
+
+    async def place_order(
+            self, symbol: str, side: str, order_type: str, amount: Decimal,
```

Comment:
I know I am an outlier here, but can you make this one long line? It fits the formatting of the rest of the library.